code/modules/spells/aoe_turf/screech.dm

```diff
@@ -63,6 +68,15 @@
 	playsound(user, 'sound/effects/creepyshriek.ogg', 100, 1)
 
 	var/datum/role/vampire/V = isvampire(user)
-	if (!V)
-		return FALSE
-	V.remove_blood(blood_cost)
\ No newline at end of file
+	if (V)
+		V.remove_blood(blood_cost)
+
+/spell/aoe_turf/screech/critfail(var/list/targets, var/mob/user)
+	user.visible_message("<span class='danger'>\The [user] emits a pathetic shriek and then falls over.</span>", "<span class='danger'>It's like a thousand needles pierce your skull.</span>")
+	user.ear_deaf = 20
+	user.stuttering = 30
+	user.Stun(5)
+	user.Jitter(150)
+	var/datum/role/vampire/V = isvampire(user)
+	if (V)
+		V.remove_blood(3*blood_cost)
```

Comment:
I think this will nuke the vampire's blood supply (120 consumed blood, 30 from casting and 90 from critical failure. Could be intended, could be unintended, if it's unintended and you just want 3x the blood amount consumed change these things to 2x, if it's intended then I argue that it's just too much blood consumed)

Reply:
Blood isn't consumed if there's a crit fail

Reply:
Pretty sure it does if the spell is still casted, which means it also enters cooldown